include/HubFramework/HUBComponentWithChildren.h

```diff
@@ -110,6 +110,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, weak, nullable) id<HUBComponentChildDelegate> childDelegate;
 
+@optional
```

Comment:
Don't use optional methods 😱 They're not compile time safe and make the API loose its explicitness. In fact, as part of https://github.com/spotify/HubFramework/issues/70 we will start linting for `@optional` and fail the build if they're used. Instead, either make it a required method (if it should be strongly coupled with this API), or make a new protocol that extends `HUBComponent`.